pixel.js

```diff
@@ -0,0 +1,87 @@
+// create grid; takes two arguments, both numbers are units of pixel boxes
+const grid = document.querySelector('.grid');
+const horizontalPixels = 40;
+const verticalPixels = 60;
+
+function makeGrid() {
```

Comment:
A better way to design this would be for `makeGrid` to take the two arguments for the grid size, rather that rely on the grid size variable being available in the parent scope.